src/modules/claims/claims.service.ts

```diff
@@ -113,6 +123,30 @@ export class ClaimsService {
   async init() {
     const chainId = this._signerService.chainId;
     this._claimManager = chainConfigs()[chainId].claimManagerAddress;
+
+    const signer = this._signerService.signer;
+    const provider = signer.provider;
+    const publicKey = await this._signerService.publicKey();
+    let ewSigner: EwSigner;
```

Comment:
why not export it from did-registry.service?

Reply:
Done